definitions/axios_v0.11.x/flow_>=v0.23.x/axios_v0.11.x.js

```diff
@@ -0,0 +1,57 @@
+declare module 'axios' {
+  declare interface AxiosXHRConfigBase<T> {
+    headers?: Object;
+    maxContentLength?: number;
+    params?: Object;
+    responseType?: 'arraybuffer' | 'blob' | 'document' | 'json' | 'text' | 'stream';
+    transformReponse?: <U>(data: T) => U;
+    transformRequest?: <U>(data: T) => U|Array<<U>(data: T) => U>;
+    validateStatus?: ?(status: number) => bool,
+    withCredentials?: boolean;
+    xsrfCookieName?: string;
+    xsrfHeaderName?: string;
+  }
+  declare interface AxiosXHRConfig<T> extends AxiosXHRConfigBase<T> {
+    data?: T;
+    method?: string;
+    url: string;
+  }
+  declare class AxiosXHR<T> {
+    config: AxiosXHRConfig<T>;
+    data: T;
+    headers: Object;
+    status: number;
+    statusText: string,
+    request: mixed, //this is the request object, not really typed currently.
+  }
+  declare class AxiosInterceptorIdent extends String {}
+  declare class AxiosRequestInterceptor<T> {
+    use(
+      successHandler: ?(response: AxiosXHRConfig<T>) => Promise<AxiosXHRConfig> | AxiosXHRConfig,
+      errorHandler: ?(error: mixed) => mixed,
+    ): AxiosInterceptorIdent;
+    eject(ident: AxiosInterceptorIdent): void;
+  }
+  declare class AxiosResponseInterceptor<T> {
+    use(
+      successHandler: ?(response: AxiosXHR<T>) => mixed,
+      errorHandler: ?(error: mixed) => mixed,
+    ): AxiosInterceptorIdent;
+    eject(ident: AxiosInterceptorIdent): void;
+  }
+  declare class Axios {
+    constructor<T>(config: AxiosXHRConfigBase<T>): Promise<T>;
+    <T>(config: AxiosXHRConfigBase<T>): Promise<T>;
+    [key: $Enum<Axios>]: Function;
```

Comment:
Ok, I'm not too familiar with this library so you understand this libdef best -- but if this isn't strictly necessary I think we should remove it. It seems like it could have potential to stir Flow in the wrong direction when mixed with the other members here